### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by nvatuan**
Init Project
- [x] Init new Rails project
- [x] Add `/test/`, config files (credentials, database, storage), `.browserslistrc` to `.gitignore`
- [x] Setup Framgia-CI on Local

image

### CODE REVIEW

**Comment by nvatuan** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,64 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.7.1'
```
Comment (vanvtt-0952) [inline Gemfile]: Sử dụng nháy đôi nhé bé
Self check file này nè

Comment (nvatuan) [reply Gemfile]: fixed

**Comment by nvatuan** [comment]
merged

**Pull request merged**